src/jaxsim/math/joint_model.py

```diff
@@ -255,7 +255,7 @@ def compute_R() -> tuple[jtp.Matrix, jtp.Array]:
         axis = jnp.array(joint_axis).astype(float).squeeze()
 
         pre_H_suc = jaxlie.SE3.from_rotation(
-            rotation=jaxlie.SO3.from_matrix(Rotation.from_axis_angle(vector=s * axis))
+            rotation=Rotation.from_axis_angle(vector=s * axis)
```

Comment:
Are you sure that `jaxlie.SE3.from_rotation` accepts a plain jax numpy vector and not a SO3 object? Because `Rotation.from_axis_angle` should return a jax numpy vector.

Reply:
Addressed in https://github.com/ami-iit/jaxsim/pull/212/commits/2e019dd3d6175b35e96a6d12e7b823c4ebf75dc6